src/graphql/root/query/on-chain-tx-fee-query.ts

```diff
@@ -28,7 +28,7 @@ const OnChainTxFeeQuery = GT.Field({
 
     const fee = await Wallets.getOnChainFee({
       walletId,
-      account: domainAccount,
+      account: domainAccount as Account,
```

Comment:
why not having `{ domainAccount: Account }` at the function signature above? we're looking some type guarantee with the casting here

Reply:
That type should come from the context, I plan to add the context type to all resolvers in the next PR.

Reply:
agree but I may be missing something: the casting is still unnecessary here compare to the existing code?

Reply:
We have 2 context types (one for public and one for authenticated). The default is public, so the type here is `Account | undefined`.  This is an authenticated query, so we either cast here or use the authenticated context type. I faced some issues using the authenticated context type, so I'll probably come up with a different option.

Reply:
my point is that the line above was already making `Account` a not undefined field?

https://github.com/GaloyMoney/galoy/pull/1384/files#diff-4fed0ac3ed1e683af6718bfe6cf2825f4eacedadc4dab1be89b387e8b29b8b6dR19